bindings/python/triplestore/backends/ontopy.py

```diff
@@ -0,0 +1,197 @@
+import io
+import os
+import tempfile
+import warnings
+from typing import TYPE_CHECKING
+
+from ontopy.ontology import get_ontology, Ontology, _unabbreviate
+
+
+from dlite.triplestore import Literal
+
+if TYPE_CHECKING:  # pragma: no cover
+    from collections.abc import Sequence
+    from typing import Generator
+
+    from dlite.triplestore import Triple
+
+
+class OntopyStrategy:
+    """Triplestore strategy for EMMOntoPy.
+
+    Arguments:
+        onto: Ontology to initiate with.  Defaults to an new ontology with
+            `base_iri`.
+        load: Whether to load the ontology.
+        kwargs: Keyword arguments passed to the ontology load() method.
+    """
+    def __init__(
+            self,
+            base_iri: str,
+            onto: Ontology = None,
+            load: bool = False,
+            kwargs: dict = {}
```

Comment:
Here we require that base_iri is given. Should we not be able to get the base_iri from the ontology if we load an external one.

Reply:
Fixed